### PR TITLE
feat: Add azure blob storage support

### PR DESCRIPTION
**Pull request by Vafilor**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes onepanelio/core#

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/build.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
-	v1 "github.com/onepanelio/core/pkg"
+	"github.com/onepanelio/cli/cloud/storage"
```
Comment (rushtehrani) [inline cmd/build.go:7]: Why this change?

Comment (Vafilor) [reply cmd/build.go:7]: @rushtehrani I'm working on removing the dependency on onepanel-core. Without this change, I'd have to update onepanel-core with these changes and pull the updated version. I think I just need the ptr utils and that's it.

Comment (Vafilor) [reply cmd/build.go:7]: @rushtehrani Actually, looks like that was it. No further need for the dependency.

Comment (rushtehrani) [reply cmd/build.go:7]: got it, we'd have to eventually pull these into a shared package.

Comment (Vafilor) [reply cmd/build.go:7]: @rushtehrani Agreed.
